MetasysServices/MetasysClient.cs

```diff
@@ -124,6 +124,7 @@ public new ApiVersion Version
                 if (Spaces != null) { Spaces.Version = version.Value; }
                 if (Streams != null) { Streams.Version = version.Value; }
                 if (Trends != null) { Trends.Version = version.Value; }
+                base.Version = value;

```

Comment:
This was a miss in the original implementation.

Reply:
This should probably be

```csharp
base.Version = version.Value
```

To be consistent with all of the assignments right above this.

But I've addressed that in #148